cucumber.js

```diff
@@ -2,11 +2,11 @@ const path = require('path');
 
 module.exports = {
   default: {
-    tags: '',
+    tags: '@campaign',
```

Comment:
You should not change the cucumber configuration often, especially every MR. If you want to run scenario with a specific tag, I've shown you how to do it.